contracts/src/v0.8/ChainSpecificUtil.sol

```diff
@@ -41,4 +41,23 @@ library ChainSpecificUtil {
     }
     return 0;
   }
+
+  function getPricesInWei() internal view returns (uint256, uint256, uint256, uint256, uint256, uint256) {
+    uint256 chainid = block.chainid;
+    if (chainid == ARB_MAINNET_CHAIN_ID || chainid == ARB_GOERLI_TESTNET_CHAIN_ID) {
+      return ARBGAS.getPricesInWei();
+    }
+
+    return (0, 0, 0, 0, 0, 0);
+  }
+
+  function postToL1StaticOverheadBytes() internal view returns (uint256) {
+    uint256 chainid = block.chainid;
+    if (chainid == ARB_MAINNET_CHAIN_ID || chainid == ARB_GOERLI_TESTNET_CHAIN_ID) {
+      // see https://developer.arbitrum.io/devs-how-tos/how-to-estimate-gas#where-do-we-get-all-this-information-from
+      return 140;
+    }
+
+    return 0;
+  }
```

Comment:
should we combine these into 1 function e.g. `getL1Gas(uint32 calldataSize)`? Thinking it might be more extensible for future integrations to different L2s. e.g. having L1 static overhead bytes might be specific to arbitrum.

Reply:
I think you're right in that these could be Arbitrum specific. What would the single function `getL1Gas` look like though? The strategy in this PR is to calculate the cost to posting to L1 for the fulfillment tx in the wrapper request; doing it based on L1 gas is another approach perhaps.

Reply:
Yeah the function will calculate the cost of posting to L1. The caller of `getL1CalldataGas(uint32 calldataSize)` supplies the calldata size like so:

`uint256 costWei = (_requestGasPrice * (_gas + s_wrapperGasOverhead + s_coordinatorGasOverhead) + getL1CalldataGas(FULFILLMENT_COMPRESSED_TX_DATA_SIZE_BYTES));`

Reply:
Ah I see what you mean, I think that makes sense 👍

Reply:
Then the implementation of function will be: 

```
function getL1CalldataGas(uint32 calldataSize) internal returns (uint256) {
  if (chainid == ARB_MAINNET_CHAIN_ID || chainid == ARB_GOERLI_TESTNET_CHAIN_ID) {
    (, uint256 l1PricePerByte, , , , ) = ChainSpecificUtil.getPricesInWei();
    return 140 + l1PricePerByte * calldataSize;
  }
  return 0;
}
```

Reply:
Updated: [`5fe066c` (#9346)](https://github.com/smartcontractkit/chainlink/pull/9346/commits/5fe066c54dd999870ed952c432850f24d0f6d35a)